### PR TITLE
Removed $:<< construct from requires for platform compatibility and idiomaticity.

### PR DESCRIPTION
**Pull request by Dazzla**
@jspc edit:

(To save @Dazzla the work so we can get this merged).

Running the create consul configuration steps in ansible on the local host seems to fail because the working directory of the script isn't set to the directory the script exists in. Thsi behaviour seems to have changed in recent ansible.

While it would work were we to just set the working directory inside ansible, this more masks the issue. This approach is more idiomatic ruby.

### CODE REVIEW

File: src/lib/consul.rb [hunk 1]
```diff
@@ -96,7 +96,7 @@ def domain_names
     put('flex/enterprise/api/url', api)
     put('flex/shared/flex-enterprise/api/url', api)
     put('flex/shared/flex-enterprise/consoleUrl', api)
-    put('flex/enterprise/consoleUrl', fqdn)
+    put('flex/shared/flex-enterprise/consoleUrl', fqdn)
```
Comment (jspc) [inline src/lib/consul.rb]: this change seems out of scope. Whats the context?

**Comment by jspc** [comment]
@Dazzla could  you update the tests too?

File: src/lib/consul.rb (hunk 1 above)
Comment (jemrayfieldft) [inline src/lib/consul.rb]: @Dazzla it needs to be:

     put('flex/shared/flex-enterprise/api/url', api)
     put('flex/enterprise/consoleUrl', fqdn)
     put('flex/shared/flex-enterprise/consoleUrl', fqdn)

But you can put it in another branch if needs be.

**Comment by jspc** [comment]
@Dazzla what about the other scripts using `$:<<'./lib'`? Also: could you add context to the PR description? To make it easier to grok in the future?

File: src/set_vars [hunk 2]
```diff
@@ -3,8 +3,7 @@
 require 'optparse'
 require 'inifile'
 
-$:<< './lib'
-require 'aws'
+require_relative '../src/lib/aws'
```
Comment (jspc) [inline src/set_vars]: won't work- that files doesn't exist in docker. Should just be `lib/aws` like with consul

**Comment by Dazzla** [comment]
Updated to 'lib/aws'. I think we need to make this testable.

**Comment by jspc** [comment]
> Updated to 'lib/aws'. I think we need to make this testable.

Make what testable, sorry?

**Comment by Dazzla** [comment]
Sorry, I meant the whole process. It's not really relevant to this PR. I'll withdraw it.
